sdk/storage/azure-storage-queue/setup.py

```diff
@@ -66,14 +66,14 @@
     ]),
     python_requires=">=3.7",
     install_requires=[
-        "azure-core<2.0.0,>=1.26.0",
```

Comment:
I can't make a comment because it's not in the diff, but line #48 I left as the beta status since we aren't releasing. Just wanted to check that this is expected

Reply:
Yeah, that seems right.